### PR TITLE
Addressed all issuses from peer review

### LINKED ISSUE #75: Peer Feedback - testing `stat_summary`
I see that your `test-stat_summary.R` uses arbitrary values as input instead of objects created by your functions. The package would be more stable if you try to test with the output of your function objects as input.

### LINKED ISSUE #78: Peer Feedback - `generate_virtual_pop` arguments
In `generate_virtual_pop` documentation, description about `dist` seems a bit unclear. For example, `dnorm()`, `dpois()` will not work although it is a function of a distribution. Specifying a list of possible functions that you can use for `dist` in the error message would be super helpful

### LINKED ISSUE #81: Peer Feedback - `generate_virtual_pop` function
When I bring up documentation in R Studio, e.g. `?generate_virtual_pop`, the author name is appended to the description of the function. This doesn't seem to be the norm with other packages. I'd consider removing the author line in the source file.

### PR DESCRIPTION
**Pull request by tguo9**
#81, #78, #77, #75

### CODE REVIEW

File: R/generate_virtual_pop.R [hunk 1]
```diff
@@ -1,11 +1,10 @@
 #' Create a virtual population
 #'
 #' This function creates a population from a distribution.
-#' Author: Tao Guo
 #'
 #'@param N The numbers of samples
 #'@param var_name the variable name that we need to create
-#'@param dist the function that we are using
+#'@param dist the distribution function for random variables (starts with 'r') that we are using
```
Comment (hwilliams10) [inline R/generate_virtual_pop.R]: Could consider adding a couple of examples? Like `rnorm`, `rexp`, etc?

**Review by hwilliams10 — APPROVED** [review]
Great job, @tguo9 . Just had one minor comment. Please let me know whether you agree.

File: R/generate_virtual_pop.R (hunk 1 above)
Comment (tguo9) [reply R/generate_virtual_pop.R]: Got it!

Comment (tguo9) [reply R/generate_virtual_pop.R]: Done!

**Review by hwilliams10 — APPROVED** [review]
Thanks, looks great

**Pull request merged**